packages/ra-ui-materialui/src/list/SingleFieldList.tsx

```diff
@@ -89,12 +102,9 @@ export const SingleFieldList = (props: SingleFieldListProps) => {
                                 to={resourceLinkPath}
                                 onClick={stopPropagation}
                             >
-                                {cloneElement(Children.only(children), {
-                                    record,
-                                    resource,
-                                    // Workaround to force ChipField to be clickable
-                                    onClick: handleClick,
-                                })}
```

Comment:
This is technically a BC break, as people may use SingleFieldList with children expecting `record` as prop. However, it's not supposed to be the case since 4.0.